matrix/view.py

```diff
@@ -29,7 +30,7 @@
         False: ("fail", "\N{HEAVY BALLOT X}"),
         }
 
-STATE_SYMNOLS = {
+STATE_SYMBOLS = {
```

Comment:
looks like currently unused but wasn't removed, feel free to remove this block, it just using TEST_SYMBOLS below